packages/babel-plugin/src/__tests__/index.test.ts

```diff
@@ -114,6 +114,36 @@ describe('babel plugin', () => {
     `);
   });
 
+  it('should transform React.cloneElement css prop', () => {
+    const actual = transform(`
+      import { cloneElement } from 'react';
+      import { css } from '@compiled/react';
+
+      const MyDiv = ({ children }) => {
+        return cloneElement(children, { css: css({ fontSize: 12 }) });
```

Comment:
This is would be the full example of what we need this for—do you think it would work?
```tsx
const styles = css({ fontSize: 12 });
const MyDiv = ({ children, className }) => {
  return cloneElement(children, { css: cx(styles, props.className) });
}
```

Reply:
The other question I've got, probably for Compiled, is should this be `props.css` or `props.className` exclusively?

Reply:
I had the same question. 

I'll add a test case for the example you provided!